sqlitecluster/SQLiteNode.cpp

```diff
@@ -617,7 +619,7 @@ bool SQLiteNode::update() {
             }
         }
 
-        SINFO("Signed in to " << numLoggedInFullPeers << " of " << numFullPeers << " full peers (plus " << (_peerList.size() - numFullPeers) << " permafollowers).");
+        SINFO("Signed in to " << numLoggedInFullPeers << " of " << numFullPeers << " full peers (plus " << (_peerList.size() - numFullPeers) << " permafollowers): " << SQList(loggedInPeers));
```

Comment:
I think we want `SComposeList` here, we don't need this to work in a sql query (which quotes every string)

Reply:
Whoops yeah I agree, updated